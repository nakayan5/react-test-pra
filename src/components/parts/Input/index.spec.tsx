import { render } from "@testing-library/react";
import { Input } from "./index";

describe("Input", () => {
  test("render", () => {
    const { asFragment } = render(<Input value={"value"} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
