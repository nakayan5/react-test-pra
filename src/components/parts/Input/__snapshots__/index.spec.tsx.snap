// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button render 1`] = `
<DocumentFragment>
  <input
    class="sc-bczRLJ ekBFnZ"
    value=""
  />
</DocumentFragment>
`;

exports[`Input render 1`] = `
<DocumentFragment>
  <input
    class="sc-bczRLJ ekBFnZ"
    value="value"
  />
</DocumentFragment>
`;
